packages/vats/src/types.js

```diff
@@ -11,6 +11,7 @@
  * @property {(value: any) => string} getId
  * @property {(value: any) => boolean} has
  * @property {() => string[]} ids
+ * @property {(...path: string[]) => Promise<any>} lookup
```

Comment:
Add a justification for `any`, please. Or change to `unknown`.

Reply:
by way of example: https://github.com/Agoric/agoric-sdk/blob/master/packages/vats/src/core/types.js#L109-L113

```
 * @property {() => ERef<Record<string, any>>} getChainBundle Required for ag-solo, but deprecated in favour of getConfiguration
 *   NOTE: we use `any` rather than `unknown` because each client that wants to call a method such as
 *   `E(userBundle.bank).deposit(payment)` has to cast userBundle.bank;
 *   ideally, the cast is to some useful type. But unknown can't be cast directly to some other type;
 *   it has to be cast to any first.
```